docs/topics/api.rst

```diff
@@ -34,6 +34,13 @@ how you :ref:`configure the downloader middlewares
     :class:`scrapy.spiders.Spider` subclass and a
     :class:`scrapy.settings.Settings` object.
 
+    .. attribute:: request_fingerprinter
+
+        The generator of request fingerprints of this crawler.
```

Comment:
(Nitpick) Should we choose a different word here? I'm concerned it might be read as if we're talking about [_generators_](https://wiki.python.org/moin/Generators).